src/backend/mailbox_manager/utils/dimail.py

```diff
@@ -68,8 +68,8 @@ def get_headers(self, user_sub=None):
 
         if response.status_code == status.HTTP_403_FORBIDDEN:
             logger.error(
-                "[DIMAIL] 403 Forbidden: Could not retrieve a token,\
-                please check 'MAIL_PROVISIONING_API_CREDENTIALS' setting.",
+                "[DIMAIL] 403 Forbidden: Could not retrieve a token,"
```

Comment:
Wow, I had no idea this was valid Python syntax. Learned something new today. (Also, apologies for taking two weeks to get around to a 2nd review of this PR…)